qa/tasks/persistent_writeback_cache_thrash.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
Nit: not sure this really needs to be a new task. You can probably just write a small script (or use the built-in `exec` task) to start `timeout <random>s rbd bench --io-type write ...`

Reply:
Deleted the new task, and use `timeout`.